src/main.ts

```diff
@@ -151,6 +159,44 @@ export class Purchases {
     return Purchases.getSharedInstance();
   }
 
+  /**
+   * Initializes the Purchases SDK by prefetching some optional data.
+   * Currently only prefetching branding information. You can call this method
+   * after configuring the SDK to speed up the first call to
+   * {@link Purchases.purchase}.
+   */
+  public async initialize(): Promise<void> {
```

Comment:
This conceptually introduces a new state to the SDK which I don't fully like... Honestly, I'm thinking about renaming this to something like `preloadResources` or something like that, so it's clearer what this does... Wdyt?

Reply:
I actually think I will do this rename... I feel this name could be confusing to what this does/we're planning to do in here (prefetching some data)

Reply:
Renamed in https://github.com/RevenueCat/purchases-js/pull/169/commits/02910315d76077b7c1c86c0752b7bfdf0c49be04